test/native/src/test/java/org/apache/shardingsphere/elasticjob/test/natived/it/operation/JavaTest.java

```diff
@@ -305,4 +308,47 @@ void testShardingStatisticsAPI() {
         });
         job.shutdown();
     }
+
+    @Test
+    void testGivenOneShardingCountWhenShutdownThenTaskCanCaptureInterruptedException() throws Exception {
+        testCaptureInterruptedException(1);
+    }
+
+    @Test
+    void testGivenOneMoreShardingCountWhenShutdownThenTaskCanCaptureInterruptedException() throws Exception {
+        testCaptureInterruptedException(2);
+    }
+
+    private void testCaptureInterruptedException(int shardingTotalCount) throws Exception {
+        String jobName = "testTaskCaptureInterruptedTask";
+        final AtomicBoolean captured = new AtomicBoolean(false);
+        // start in 2 seconds
```

Comment:
- I have the impression that ShardingSphere's code constraints don't like inline comments, as Javadoc doesn't display them. Can you change the comments to Javadoc style, or remove the inline comments?